docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+x-uffizzi:
+  ingress:
+    service: application
+    port: 6006
+
+services:
+  application:
+    image: "${APP_IMAGE}"
+    ports:
```

Comment:
I don't think this does anything on Uffizzi, but it may be helpful for people reading this file. Optional.

Reply:
Yeah, it's helpful when we read this file :)